inst/report/KRIReportBySite.Rmd

```diff
@@ -93,6 +93,11 @@ study_id <- purrr::map(assessment, function(kri) {
     as.character() %>% 
     unique()
 
+## Subjects
+subjects <- purrr::map(assessment, ~length(.$lData$dfSUBJ[[Read_Mapping()$dfSUBJ$strIDCol]])) %>%
+ unlist() %>%
+ max()
+
```

Comment:
We will likely be ingesting the entire snapshot object as we continue to refactor the report, but for now I think this approach is good, but want to make sure that it matches the CTMS data when provided. 

Because of this, I'd suggest doing something like this: 

```suggestion
## Subjects
status_study_exists <- !is.null(params$status_study)

if (status_study_exists ) {
  subjects <- params$status_study[[Read_Mapping()[["dfSTUDY"]][["strActualSubjectsCol"]]]]
}

```